CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ### [master (unreleased)](https://github.com/rails-api/active_model_serializers/compare/v0.10.2...master)
 
 Breaking changes:
+- [#1791](https://github.com/rails-api/active_model_serializers/pull/1791) (@bf4, @NullVoxPopuli)  
+  - JSON API Type Transform functions independent of Key Transform.
+  - Added jsonapi_type_transform config option.
+  - Added jsonapi_namespace_separator config option.
```

Comment:
Both, I think.
and that key_transform won't affect the type's key.

The new options could be configured to behave the old way, I'm pretty sure